### DISCUSSION TITLE
https://theme-hope.vuejs.press/zh/guide/markdown/presentation.html

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#558] [Announcements]
# https://theme-hope.vuejs.press/zh/guide/markdown/presentation.html

让你的 VuePress 站点中的 Markdown 文件支持幻灯片。 我们使用 reveal.js 来支持这一功能。

https://theme-hope.vuejs.press/zh/guide/markdown/presentation.html

**Comment by cnlibo** [comment]
您好！
能否请问一下，该如何设置来实现幻灯片的自动播放？
先谢谢了！
